docs/install/2-install-server-with-podman-and-systemd.rst

```diff
@@ -0,0 +1,585 @@
+                NotifyAccess=all
+
+                [Install]
+                WantedBy=default.target
+
+        .. only:: iso
+
+            .. code-block:: 
+            :substitutions:
+
+                [Unit]
+                Description=Podman 
+                Documentation=https://docs.inmanta.com
+                Wants=network-online.target
+                After=network-online.target
+                RequiresMountsFor=%t/containers
+
+                [Service]
+                Environment=PODMAN_SYSTEMD_UNIT=%n
+                Restart=on-failure
+                TimeoutStopSec=70
+                ExecStart=/usr/bin/podman run \
+                        --cidfile=%t/%n.ctr-id \
+                        --cgroups=no-conmon \
+                        --sdnotify=conmon \
+                        -d \
+                        --replace \
+                        --publish=127.0.0.1:8888:8888 \
+                        --uidmap=993:0:1 \
+                        --uidmap=0:1:993 \
+                        --uidmap=994:994:64543 \
+                        --gidmap=993:0:1 \
+                        --gidmap=0:1:993 \
+                        --gidmap=994:994:64543 \
+                        --name=inmanta-orchestrator-server \
+                        --volume=%E/inmanta/inmanta.cfg:/etc/inmanta/inmanta.cfg:z \
+                        --volume=%E/inmanta/license/com.inmanta.license:/etc/inmanta/license/com.inmanta.license:z \
+                        --volume=%E/inmanta/license/com.inmanta.jwe:/etc/inmanta/license/com.inmanta.jwe:z \
+                        --volume=%h/.local/share/inmanta-orchestrator-server/logs:/var/log/inmanta:z \
+                        --entrypoint=/usr/bin/inmanta \
+                        --user=993:993 \
+                        containers.inmanta.com/containers/service-orchestrator:|version_major| \
+                        --log-file /var/log/inmanta/server.log --log-file-level 2 --timed-logs server
+                ExecStop=/usr/bin/podman stop \
+                        --ignore -t 10 \
+                        --cidfile=%t/%n.ctr-id
+                ExecStopPost=/usr/bin/podman rm \
+                        -f \
+                        --ignore -t 10 \
+                        --cidfile=%t/%n.ctr-id
+                Type=notify
+                NotifyAccess=all
+
+                [Install]
+                WantedBy=default.target
+
+
+        You can paste this configuration in a file named ``inmanta-orchestrator-server.service`` in the systemd folder for your user.
+        This folder is typically ``~/.config/systemd/user/``.
+
+    .. tab-item:: Rootful setup
+        :sync: rootful-setup
+
+        .. only:: oss
+
+            .. code-block:: 
+
+                [Unit]
+                Description=Podman 
+                Documentation=https://docs.inmanta.com
+                Wants=network-online.target
+                After=network-online.target
+                RequiresMountsFor=%t/containers
```

Comment:
@bartv this will not work as the user of the unit is root and not inmanta.  I am not sure how to work around this, should we have two unit files? The root one starting the inmanta one? And the inmanta one starting the container?

Reply:
Same issue for `--cidfile=%t/%n.ctr-id`

Reply:
My question is rather: what is the sys-admin way of doing this?